### DISCUSSION TITLE
What is CX from network

### DISCUSSION THREAD

**Discussion by kamontat** [nicolargo/glances#2221] [General]
I cannot find any information on CX even on [document][doc], they just skip the explanation.

[doc]: https://glances.readthedocs.io/en/latest/api.html#get-network

**Comment by RazCrimson** [comment]
@kamontat 
My quick guess would be the combined rate (cx = rx + tx)
The same would extend for the cumulative fields too.

**Comment by nicolargo** [comment]
Yes, this is exactly that @RazCrimson . 

The "c" is for "cumulative". 

I am going to transfert thsi conversation as an issue in order to add the description in the API documentation.

Thanks for the report @kamontat
